_layouts/help_landing.html

```diff
@@ -7,7 +7,7 @@
   {% include wip.html %}
   {% include skip_nav.html %}
   {% include banner.html %}
-  {% include language_search.html logo=true signin=false contact=true %}
+  {% include header--help.html %}
```

Comment:
Our other partials use underscores, what was the motivation for dashes?

Reply:
> Our other partials use underscores, what was the motivation for dashes?

I'm not especially attached to it, but there was one example of precedent here:

https://github.com/18F/identity-site/blob/demo/_includes/hero--search.html

As I understand and applied it here, the idea for it could be as a [BEM](http://getbem.com/)-like modifier, as this is intended to serve as an in-place variant substitute for the base `header.html` partial.

I'm not sure how often these sorts of cases would arise, since in many we could simply parameterize the partial to change its behavior. In this case, since there's enough differences between the two variants, it probably would have been more work to handle the logic for parameters than to create an entirely different partial. That being said, I'd not want to set a convention here if it's not something we generally want to encourage.

Reply:
ok! As long as this it's not the only one, I guess that's fine?

Reply:
Thanks for the clarification @cannandev ! I agree that we can simplify a bit to remove `language_search.html`. Per your suggestion, I did this in 7539b80.